pages/_app.js

```diff
@@ -13,7 +14,7 @@ export default function App({ Component, pageProps }) {
 
   const bookCounter = books.length;
 
-  function editBook(currentBook) {
+  function handleEditBook(currentBook) {
```

Comment:
The submit functions themselves look good to me, but I don't understand why you place them in your `_app.js` file.
Can you explain to me, please?